_includes/google-tag-manager.html

```diff
@@ -0,0 +1,6 @@
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
nit: is there an associated jira for this, lets name the branch as per the convention

Reply:
I've added it in the description! not changing the branch name because the build works strangely with any slashes in the branch name